pkg/worker/scm/provider/git.go

```diff
@@ -66,16 +98,16 @@ func (g *Git) Clone(url, ref, destPath string) (string, error) {
 			return "", err
 		}
 
-		outputs = outputs + string(output) + "/n"
+		outputs = outputs + string(output) + "\n"
```

Comment:
Good catch, but the newline is not needed, will be removed by https://github.com/caicloud/cyclone/pull/410/files#diff-4f920adfd8e1c019748c21696264f04e